src/gc/gc.d

```diff
@@ -1350,6 +1350,15 @@ immutable uint[B_MAX] binsize = [ 16,32,64,128,256,512,1024,2048,4096 ];
 immutable size_t[B_MAX] notbinsize = [ ~(16-1),~(32-1),~(64-1),~(128-1),~(256-1),
                                 ~(512-1),~(1024-1),~(2048-1),~(4096-1) ];
 
+alias PageBits = GCBits.wordtype[PAGESIZE / 16 / GCBits.BITS_PER_WORD];
+static assert(PageBits.length * GCBits.BITS_PER_WORD * 16 == PAGESIZE);
+
+private void set(ref PageBits bits, size_t i) @nogc pure nothrow
```

Comment:
IMO defining this function is too verbose. Why not just call `bts(toFree.ptr, i)` below?